README.md

```diff
@@ -16,12 +16,42 @@
 
 - Windows 10 (64-bit)
 - MacOS
```

Comment:
Is there a MacOS minimum version?

Reply:
The MacOS minimum version is based on the Unreal version. I can add that in another PR.